guide/pgfsys-tex4ht.def

```diff
@@ -314,7 +314,7 @@
   \hbox to#2\bgroup
     \hsize=#2\relax
     \vbox\bgroup    
-% though it works, I'm desabling the above mentioned hack to make tex4ht behave with <p> <\p>, because it blurps the positioning 
+% though it works, I'm disabling the above mentioned hack to make tex4ht behave with <p> <\p>, because it blurps the positioning 
```

Comment:
Changes in this file may better be reported at their original place, and we might want to update this as well after so many years (and check if it works better now...).